modules/cpmso-failure-domains-provider.adoc

```diff
@@ -21,6 +21,11 @@ The control plane machine set concept of a failure domain is analogous to existi
 |X
 |link:https://cloud.google.com/compute/docs/regions-zones[zone]
```

Comment:
Could be out of scope but this should state `Regions and zones`.

Reply:
IIRC, we are only referring to zones here, but the GCP docs page has both on one page